workers/unity/Packages/io.improbable.gdk.core/Subscriptions/Reader.cs

```diff
@@ -17,7 +17,9 @@ public abstract class Reader<TComponent, TUpdate> : IRequireable
         protected readonly EntityManager EntityManager;
         protected readonly Entity Entity;
         protected readonly EntityId EntityId;
+
         private static readonly uint ComponentId = ComponentDatabase.GetComponentId<TComponent>();
+        private static readonly ComponentType ComponentAuthType = ComponentDatabase.Metaclasses[ComponentId].Authority;
```

Comment:
Use `ComponentDatabase.GetMetaclass(componentId)` instead? It gives a more helpful error if this ever was to throw an error